src/inference/dev_api/openvino/runtime/internal_properties.hpp

```diff
@@ -13,6 +13,14 @@
 
 namespace ov {
 
+namespace internal {
+/**
+ * @brief Read-only property to get a std::vector<PropertyName> of supported read-only properties.
```

Comment:
`std::vector<PropertyName> of supported read-only properties.` 
I think supported properties could have other mutability besides RO (config_device_id is WO)

Also it would be good to describe the scope of properties returned by this query, i.e. if it's unique set of internal properties or combination of external and internal ones.

Reply:
> `std::vector<PropertyName> of supported read-only properties.` I think supported properties could have other mutability besides RO (config_device_id is WO)
> 
> Also it would be good to describe the scope of properties returned by this query, i.e. if it's unique set of internal properties or combination of external and internal ones.

`read-only` was replaced by `internal`